WordPress/twentytwentyone
# Use IE11 ponyfill

This PR tries an alternative to using postcss to create IE specific stylesheets. It still makes use of the WP global `$is_IE` (some discussion in #8).

The benefits of this approach are:
- Doesn't require us to bundle IE front-end and editor specific stylesheets with the theme
- Reduces the build time by removing the postcss build process for IE stylesheets  

h/t @maggiecabrera @scruffian for implementing this in Seedlet.

**To test**
- Check out this PR
- Visit the site on IE 
- Verify that the theme styles are basically supported
